src/components/[guild]/RolePlatforms/components/AddRoleRewardModal/components/AddFormPanel/AddFormPanel.tsx

```diff
@@ -64,18 +64,23 @@ const AddFormPanel = ({ onAdd }: AddRewardPanelProps) => {
         <Stack spacing={6}>
           <ContinueWithExistingFormAlert {...{ onAdd }} />
           <CreateFormForm />
-          <Button
-            colorScheme="green"
-            rightIcon={<ArrowRight />}
-            w="max-content"
-            ml="auto"
-            onClick={methods.handleSubmit(onSubmit, console.error)}
-            loadingText="Creating form"
-            isLoading={isLoading}
-            isDisabled={!fields?.length}
-          >
-            Create form & continue
-          </Button>
+          <Flex mt="4" justifyContent="space-between" alignItems="end">
```

Comment:
I think we should display the text and button below each other on smaller screens, wdyt? Maybe the text should be below the button actually in that case.